lib/matplotlib/tests/test_axes.py

```diff
@@ -1476,6 +1476,15 @@ def _as_mpl_axes(self):
         'Expected a PolarAxesSubplot, got %s' % type(ax)
     plt.close()
 
+    # test focusing of Axes in other Figure
+    fig1, ax1 = subplots()
```

Comment:
Indeed `test_as_mpl_axes_api` seems to be for customizations with the `_as_mpl_axes` function.  I will add the new function.